src/main/scala/mg7/loquats/7.stats.scala

```diff
@@ -47,6 +42,7 @@ case object statsDataProcessing extends DataProcessingBundle()(
     LazyTry {
       val csvWriter = csv.newWriter(statsCSV)
 
+      // TODO the whole method is a bit primitive
```

Comment:
for a start, do something about https://github.com/ohnosequences/mg7/pull/84/files/c9945b3dce720a26183b603d777db5e166c2553f#diff-23c521293b340b995ae334c38fb6ce62R50